pkg/config/mq_sink_protocol.go

```diff
@@ -58,7 +58,7 @@ func (p *Protocol) FromString(protocol string) error {
 	case "open-protocol":
 		*p = ProtocolOpen
 	default:
-		return cerror.ErrMQSinkUnknownProtocol.GenWithStackByArgs(protocol)
+		return cerror.ErrSinkUnknownProtocol.GenWithStackByArgs(protocol)
```

Comment:
Should we rename this to `sink_protocol`?

Reply:
Or message protcol?

Reply:
`sink_protocol` is ok. I will modify it later.